src/Microsoft.IdentityModel.Tokens/SupportedAlgorithms.cs

```diff
@@ -274,6 +277,28 @@ internal static bool IsSupportedAuthenticatedEncryptionAlgorithm(string algorith
             return false;
         }
 
+        internal static bool IsSupportedAesGcmEncryptionAlgorithm(string algorithm, SecurityKey key)
```

Comment:
Since this is internal and static, i don't think we need two methods. We could have: IsSupportedEncryptionAlgorithm.

Reply:
If we stick with two methods, not recommended, this method should come before: IsSupportedAuthenticatedEncryptionAlgorithm.

Reply:
Addressed it in [0761e11d8bbb8920a1e0716c7df7faf7d15c7187](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1606/commits/0761e11d8bbb8920a1e0716c7df7faf7d15c7187)